packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -164,11 +164,12 @@
 }
 
 .wp-block-legacy-widget {
-	.components-base-control {
+	.wp-block-legacy-widget__selector-container {
 		width: 100%;
 	}
-	.components-select-control__input {
```

Comment:
This CSS was removed because it wasn't being used (overwritten by other styles).